account_analytic_project/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Project and analytic account integration
+========================================
+
+Gives the option to create projects directly from the analytic accounts they
+should be linked to.
+
+Without this module, if you first create an analytic account and later on you
+decide to have a project as well for this account, the situation is very
+difficult to rectify.
+
```

Comment:
I am not saying the module is essential under 8.0 as it was under previous releases. But having a checkbox "use tasks"  that does the work of a "create project" button, is nice for the initiated, not for the average user that already struggles to find his or her way in Odoo.